snapcraft.yaml

```diff
@@ -0,0 +1,20 @@
+name: lxdock
+version: git
+summary: lxdock
+description: |
+  Build and orchestrate your development environments with LXD 
+  - a.k.a. Vagrant is Too Heavy
+base: core18
+confinement: strict
+
+parts:
+  lxdock:
+    plugin: python
+    python-version: python3
+    source: .
+    requirements: requirements-dev.txt
```

Comment:
am I reading this right? do the dev requirements end up in the snap? why? we don't need them in the snap or any other kind of binary distribution like deb/rpm

Reply:
FIxed: 5e261b3